test.js

```diff
@@ -27,7 +27,7 @@ test('records screen', async t => {
   const recorder = aperture();
   t.true(fs.existsSync(await recorder.startRecording()));
   await delay(1000);
-  const videoPath = await recorder.stopRecording();
+  const {filePath: videoPath} = await recorder.stopRecording();
```

Comment:
You modified the incorrect method. It's `startRecording()` you need to modify, on line 28.

Reply:
Oops.